forge-gui/res/cardsfolder/upcoming/inga_and_esika.txt

```diff
@@ -0,0 +1,10 @@
+Name:Inga and Esika
+ManaCost:2 G U
+Types:Legendary Creature Human God
+PT:4/4
+T:Mode$ SpellCast | ValidCard$ Creature | CheckSVar$ X | SVarCompare$ GE3 | ValidActivatingPlayer$ You | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell, if three or more mana from creatures was spent to cast it, draw a card.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddAbility$ AnyMana | AddKeyword$ Vigilance | Description$ Creatures you control have vigilance and "{T}: Add one mana of any color. Spend this mana only to cast a creature spell."
+SVar:AnyMana:AB$ Mana | Cost$ T | Produced$ Any | RestrictValid$ Spell.Creature | Amount$ 1 | SpellDescription$ Add one mana of any color. Spend this mana only to cast a creature spell.
+SVar:X:TriggeredCard$CastTotalManaSpent Creature
```

Comment:
TriggeredObjects can never be checked like that on the event itself
Needs to be done like on Vazi

Reply:
Do you mean like this ? currently not working though

Reply:
yea still needs some support in ForgeScript